manifests/00-ingress-credentials-request.yaml

```
@@ -10,6 +10,7 @@ metadata:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Ingress
```

This manifest already has `capability.openshift.io/name: CloudCredential`, so you'll need to add "+Ingress" to the existing value:
```suggestion
    capability.openshift.io/name: CloudCredential+Ingress
```

---

Somehow that got lost between all the rebases. Let me readd.

---

Done.